test/e2e/fleetautoscaler_test.go

```diff
@@ -221,3 +223,108 @@ func getAllocation(f *v1alpha1.Fleet) *v1alpha1.FleetAllocation {
 		},
 	}
 }
+
+//Test fleetautoscaler with webhook policy type
+// scaling from Replicas equals to 1 to 2
+func TestAutoscalerWebhook(t *testing.T) {
+	t.Parallel()
+	pod, svc := defaultAutoscalerWebhook()
+	pod, err := framework.KubeClient.CoreV1().Pods(defaultNs).Create(pod)
+	if assert.Nil(t, err) {
+		defer framework.KubeClient.CoreV1().Pods(defaultNs).Delete(pod.ObjectMeta.Name, nil) // nolint:errcheck
+	} else {
+		// if we could not create the autoscaler, their is no point going further
+		logrus.Error("Failed creating autoscaler, aborting TestAutoscalerBasicFunctions")
```

Comment:
`logrus.Error` and return - this could be `assert.FailNow()`:

```suggestion
		assert.FailNow(t, "Failed creating autoscaler, aborting TestAutoscalerBasicFunctions")
```

(And remove the `return` statement)

Same on below as well.

Reply:
Thanks Mark, I have applied the patch. That PullAlways parameter should help.